<launch>
    <machine name="local" address="localhost" default="true"/>
    <node name="Chisel" pkg="chisel_ros" type="ChiselNode" output="screen"> 
        <param name="chunk_size_x" value="16"/>
        <param name="chunk_size_y" value="16"/>
        <param name="chunk_size_z" value="16"/>
        <param name="truncation_scale" value="10.0"/>
        <param name="integration_weight" value="1"/>
        <param name="use_voxel_carving" value="true"/>
        <param name="use_color" value="false"/>
        <param name="carving_dist_m" value="0.05"/>
        <param name="voxel_resolution_m" value="0.02"/>
        <param name="far_plane_dist" value="1.5"/>
        <param name="base_transform" value="map"/>
        <param name="depth_image_transform" value="sim_camera"/>
        <param name="color_image_transform" value="sim_camera"/>
        <param name="fusion_mode" value="DepthImage"/>
    
        <remap from="/depth_image" to="/sim_camera/depth"/>
        <remap from="/depth_camera_info" to="/sim_camera/camera_info"/>
        <remap from="/color_image" to="/sim_camera/color"/>
        <remap from="/color_camera_info" to="/sim_camera/camera_info"/>
        <remap from="/camera/depth_registered/points" to="/sim_camera/depth/points"/>

    </node>
</launch>
